modules/ROOT/pages/index.adoc

```diff
@@ -62,6 +61,7 @@ OpenSSL is a required dependency for the driver.
 . Verify that Windows has installed SSL, and registered the path of the binary.
 Open the command prompt (`cmd`), and enter the following:
 +
+[source, shell]
```

Comment:
strictly, these should be **console** (or **terminal**),
**shell** - to the JS highlighter library - is a pseudonym for **bash**, but bash-as-scripting, not the command line.